--- lxqt-session-0.8.0/CMakeLists.txt.unify	2014-12-20 12:26:58.315251067 -0200
+++ lxqt-session-0.8.0/CMakeLists.txt	2014-12-20 12:27:06.260282190 -0200
@@ -17,7 +17,7 @@
     find_package(Qt5X11Extras REQUIRED QUIET)
     find_package(Qt5LinguistTools REQUIRED QUIET)
 
-    find_package(lxqt-qt5 REQUIRED QUIET)
+    find_package(lxqt REQUIRED QUIET)
     include(${LXQT_USE_FILE})
 
     find_package(Qt5Xdg REQUIRED QUIET)
